gen3/lib/aws.sh

```diff
@@ -194,7 +194,7 @@ vpc_name="$GEN3_WORKSPACE"
 #
 vpc_octet2=GET_A_UNIQUE_VPC_172_OCTET2
 vpc_octet3=GET_A_UNIQUE_VPC_172_OCTET3
-squid-nlb-endpointservice-name=CSOC_SQUID_NLB_SERVICE_NAME
+squid-nlb-endpointservice-name="CSOC_SQUID_NLB_SERVICE_NAME"
```

Comment:
This hides the variable default in `variables.tf` - put the "com.amazonaws.vpce.us-east-1.vpce-svc-0ce2261f708539011" value here too, or just don't put the variable in `config.tfvars`

Reply:
I removed it from the config.tvars